packages/react/src/Tooltip/Tooltip.module.css.d.ts

```diff
@@ -0,0 +1,7 @@
+declare const styles: {
+  readonly "Tooltip": string;
+  readonly ":popover-open": string;
+  readonly "tooltip-appear": string;
+};
+export = styles;
+
```

Comment:
Think this file will need to be regenerated 🤔 

`npm run build:lib` might do it

Reply:
weird, could you please try `build:css-types` script in `packages/react`?